ci-operator/config/opendatahub-io/caikit-nlp/opendatahub-io-caikit-nlp-release.yaml

```diff
@@ -0,0 +1,46 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: caikit-nlp
+promotion:
+  namespace: opendatahub-io
+  tag: stable
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.12"
```

Comment:
@Xaenalt @heyselbi @rpancham  do you know if setting `version: "4.12"` is alright here?